examples/bfetch_explorer/tsconfig.json

```diff
@@ -11,5 +11,8 @@
     "server/**/*.ts",
     "../../typings/**/*",
   ],
-  "exclude": []
+  "exclude": [],
+  "references": [
+    { "path": "../../src/core/tsconfig.json" }
```

Comment:
not a hard requirement, but it reduced the number of processed files

`node  --max-old-space-size=6144 ./node_modules/.bin/tsc -p examples/bfetch_explorer/tsconfig.json  --extendedDiagnostics --noEmit`

with references
```
Files:                         876
Lines:                      219410
Nodes:                      789781
Identifiers:                286371
```

without references 
```
Files:                        1203
Lines:                      272257
Nodes:                      930313
Identifiers:                333766
```